src/holodeck/agents.py

```diff
@@ -113,6 +116,34 @@ def set_control_scheme(self, index):
         self._current_control_scheme = index % self._num_control_schemes
         self._control_scheme_buffer[0] = self._current_control_scheme
 
+    def get_control_scheme_min_values(self, control_scheme):
+        """
+        Returns the minimum values for the given control scheme
+        Args:
+            control_scheme:
+
+        Returns:
+            A list of minimum values for the given control scheme. Each entry corresponds to the max value
+            of the corresponding entry (e.g roll or pitch) in the control scheme.
+
+            The list will contain None values where there are no minimum values
+        """
+        raise NotImplementedError('Must be implemented by the child agent')
+
+    def get_control_scheme_max_values(self, control_scheme):
+        """
+        Returns the maximum values for the given control scheme
+        Args:
+            control_scheme:
+
+        Returns:
+            A list of maximum values. Each entry corresponds to the max value
+            of the corresponding entry (e.g roll or pitch) in the control scheme
+
+            The list will contain None values where there are no maximum values
+        """
+        raise NotImplementedError('Must be implemented by the child agent')
+
```

Comment:
Let's move these new methods into the `ContinuousActionSpace` class. Maybe call them `get_low_values` and `get_high_values`?

Reply:
Changed to get_low and get_high from your suggestion below.